src/ServiceBusChannel.php

```diff
@@ -24,30 +24,6 @@ class ServiceBusChannel
     protected $useStaging = false;
     protected $ventureConfig = [];
 
-    /**
-     * ServiceBusChannel constructor.
-     *
-     * @param array $ventureConfig
-     */
-    public function __construct(array $ventureConfig = [])
-    {
-        $ventureConfigVars = [
-            'services.service_bus.username',
-            'services.service_bus.password',
-            'services.service_bus.venture_config_id',
-            'services.service_bus.enabled',
-            'services.service_bus.endpoint',
-        ];
-
-        foreach ($ventureConfigVars as $name) {
-            $this->ventureConfig[$name] = isset($ventureConfig[$name]) ? $ventureConfig[$name] : config($name);
-        }
-
-        $this->client = new Client([
```

Comment:
I'm a little concerned that this won't be done in the constructor anymore. Not sure if this will impact anything else? 
Previously, we could rely on the client always being available because it was assigned in the constructor

Reply:
@norbybaru @SmileyJoe thoughts? 
I'll look deeper when I'm out my meetings.

Reply:
cool. this should not affect anything. 
I have moved this to line [line 53](https://github.com/RingierIMU/service-bus-notifications-channel/pull/11/files#diff-28c363f5a8e835caecbc96370cfc3e9bR53)
any uses of `$this->client` are after that

Reply:
After looking through the process logic, this is definitely a better approach. I prefer getting a new client each time just to ensure that the config changes properly.

Reply:
Is there any good reason why this we removed it from Constructor? I am happy to have it on the constructor reason being is it make its clear that there is initialisation of some config rather than having it on the `send()` function
. the `send()` should preferably only do what it should do which sending and rather not doing config initialisation and sending. Having this on the `constructor()` will make more sense unless there is a special that need this to be done on the `send()` function.

Reply:
@norbybaru this has been moved be a in the `send` because we can not pass anything in the contractor.
this change to setting `$this->config` allows for the passing of venture configs that can then be used in setting this up